### DISCUSSION TITLE
Make hit box on mobile span the entire screen.

### DISCUSSION THREAD

**Discussion by kevin8999** [ppy/osu#31070] [Ideas] [answered]
When playing osu!mania on mobile, the hit box to "hit" a note is just a square.

![Screenshot_20241210_184934](https://github.com/user-attachments/assets/efc62d85-8466-4e8e-8098-2edfbf5b9bbd)

However, I would prefer if it spanned the entire lane such as in this image below.

![Screenshot_20241210_184755_Roblox](https://github.com/user-attachments/assets/dbe4cdbf-c9b5-44aa-bddb-a70f0a6779b4)

I apologize if the hit box is hard to see, but essentially, the area where one can "hit" a note spans the entire device's height.

Also, in case you're curious, the image above is a Roblox game called [rosu!mania](https://www.roblox.com/games/9051087354/rosu-mania-7-50-1-Music-Rhythm).

This change would be a nice improvement as my fingers sometimes tap outside of this hit box.

**Comment by iveurne** [comment]
according to other questions touch versatility and skinning of mania touch controls is being worked on

**Reply by kevin8999** [reply]
Oh, that's nice to hear! Do you have the link to the discussion?

**Reply by iveurne** [reply, marked as answer]
https://github.com/ppy/osu/discussions/30769#discussioncomment-11443659

it just says is happening in the future not exactly being worked on but oh well better than nun

**Comment by Gcgg1** [comment]
Is this request implemented alrdy?

**Reply by peppy** [reply]
Please check the settings for touch options:

![IMG_8179](https://github.com/user-attachments/assets/e58df9a6-1044-413c-9e9e-da9313f612ea)
